WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java

```diff
@@ -282,6 +277,10 @@ public boolean getLocation() {
         return mSettings.location;
     }
 
+    public boolean getOptmizeImage() {
```

Comment:
Small spelling mistake here, should be `getOptimizeImage`.

Better yet, we could use `optimizedImage` as the property and `getOptimizedImage` for the getter, but that's not a big deal.